assets/js/blocks/cart-checkout/mini-cart-contents/editor.scss

```diff
@@ -0,0 +1,18 @@
+.wp-block-woocommerce-mini-cart-contents {
+	max-width: 512px;
+	margin: 0 auto;
+}
+
+.wp-block-woocommerce-filled-mini-cart-contents-block {
+	.block-editor-block-list__layout {
+		display: flex;
+		flex-direction: column;
+		min-height: calc(100vh - 2 * $gap-largest);
+	}
+}
+
+.wp-block-woocommerce-mini-cart-products-table-block {
+	flex-grow: 1;
+	flex-shrink: 1;
```

Comment:
Can we remove these two declarations? `margin-bottom: auto` seemed to be enough to get the correct alignment.

Reply:
You're right, in the editor context, `margin-bottom` is enough.